apps/fabric-website/src/components/Nav/Nav.tsx

```diff
@@ -28,37 +39,80 @@ export class Nav extends React.Component<INavProps, INavState> {
   }
 
   private _renderLink(page: INavPage, linkIndex: number): React.ReactElement<{}> {
-    const childLinks = page.pages ? this._renderLinkList(page.pages, true) : null;
     const ariaLabel = page.pages ? 'Hit enter to open sub menu, tab to access sub menu items.' : '';
     const title = page.title === 'Fabric' ? 'Home page' : page.title;
+    const childLinks = page.pages ? this._renderLinkList(page.pages, true) : null;
 
     return (
-      <li
-        className={css(
-          styles.link,
-          _isPageActive(page) ? styles.isActive : '',
-          _hasActiveChild(page) ? styles.hasActiveChild : '',
-          page.isHomePage ? styles.isHomePage : '',
-          page.className ? styles[page.className] : ''
-        )}
-        key={linkIndex}
-      >
-        {page.isUhfLink && location.hostname !== 'localhost' ? (
-          ''
-        ) : (
-          <a href={page.url} onClick={this.props.onLinkClick} title={title} aria-label={ariaLabel}>
-            {page.title}
-          </a>
-        )}
-
-        {childLinks}
-      </li>
+      <span>
```

Comment:
hmm. This doesn't quite make sense.  a `span` probably shouldn't be wrapping an `li`.